prow/plugins/slackevents/slackevents_test.go

```diff
@@ -71,6 +71,21 @@ func TestPush(t *testing.T) {
 	pushEvManual.Pusher.Name = "Jester Tester"
 	pushEvManual.Pusher.Email = "tester@users.noreply.github.com"
 	pushEvManual.Sender.Login = "tester"
+	pushEvManual.Ref = "refs/head/master"
+
+	pushEvManualBranchWhiteListed := pushEv
+	pushEvManualBranchWhiteListed.Pusher.Name = "Warren Teened"
+	pushEvManualBranchWhiteListed.Pusher.Email = "wteened@users.noreply.github.com"
+	pushEvManualBranchWhiteListed.Sender.Login = "wteened"
```

Comment:
bikeshed: we used the "tester" user before. Where does wteened come from?

Reply:
The idea here was to separate test cases as much as possible: Test "tester" as completely absent from any whitelist. Test "wteened" as unable to push to master, but able to push to his own q-"warren-teened" branch, "warrens-branch".